src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngineResolver.cs

```diff
@@ -29,7 +29,7 @@ internal class RecalcEngineResolver : SimpleResolver
             RecalcEngine parent,
             PowerFxConfig powerFxConfig,
             RecordType parameters)
-            : base(powerFxConfig.EnumStore.WithRequiredEnums(GetFullFunctionLibrary(powerFxConfig.ExtraFunctions.Values.ToArray())).EnumSymbols, powerFxConfig.ExtraFunctions.Values.ToArray())
```

Comment:
>GetFullFunctionLibrary [](http://example.com/codeflow?start=61&length=22)

Calling GetFullFunctionLibrary() is problematic. It's pulling in functions that the interpreter doesn't use.

Reply:
removed the call.